scripts/flake_checker.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python execution for checking whether the tests output is flaky."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+
+import python_utils
+
+import requests
+
+FLAKE_REPORT_URL = 'https://oppia-flake-report.herokuapp.com/flake-report'
```

Comment:
I can define this URL as an env variable in CI but I'll have to copy this in all the github/workflow files. Let me know if you know a better way to handle this?

Reply:
I think this is fine